packages/markdown/components/Image/index.jsx

```diff
@@ -1,18 +1,52 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
 /* eslint-disable react/jsx-props-no-spreading */
 
 const React = require('react');
 const PropTypes = require('prop-types');
 
-const Image = props => {
-  const { align, title, alt, width, height } = props;
-  const extras = { align, width, height };
-  return <img {...props} alt={alt} title={title} {...extras} loading="lazy" />;
-};
+class Image extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { lightbox: false };
+  }
+
+  render() {
+    const { align, title, alt, width, height, className = '' } = this.props;
+    const extras = { align, width, height };
+    return (
+      <React.Fragment>
+        <img
+          {...this.props}
+          alt={alt}
+          title={title}
+          {...extras}
+          loading="lazy"
+          onClick={() => className !== 'emoji' && this.setState({ lightbox: true })}
+        />
+        {className === 'emoji' || (
+          <span
+            className="lightbox"
+            onClick={() => this.setState({ lightbox: false })}
```

Comment:
Wanna use hooks for this haha? 😬

Reply:
Yo absolutely not. You ever try to use hooks and `npm link` from a Lerna repo at the same time?

Reply:
[kills self](https://reactjs.org/warnings/invalid-hook-call-warning.html#mismatching-versions-of-react-and-react-dom)

Reply:
> 🔴 Do not call Hooks in class components.

...

![consciousness-709143-400x300](https://user-images.githubusercontent.com/33762/80436521-31d8bd00-88b4-11ea-8281-04924129e261.jpg)

Reply:
### 🙃